CONTRIBUTING.md

```diff
@@ -7,14 +7,16 @@ Thanks for your interest in contributing to Turbo!
 - [Contributing to Turbo](#contributing-to-turbo)
   - [Contributing to Turborepo](#contributing-to-turborepo)
     - [Building Turborepo](#building-turborepo)
+    - [TLS Implementation](#tls-implementation)
```

Comment:
some vscode magic extension added these in. I guess it recognized that this is a ToC and noticed missing headings. Pretty cool